### PR TITLE
LPS-99488 Make assetTitleMap a JSON object to retrieve the proper translation within syncUI

### PR DESCRIPTION
**Pull request by dacousalr**
Hi @orsolyalukacs, could you check this solution? Please, tell me if there is anything that needs further clarification. Thanks

### CODE REVIEW

**Comment by dacousalr** [comment]
ci:test:sf

**Comment by dacousalr** [comment]
ci:test:relevant

**Review by orsolyalukacs — COMMENTED** [review]
Just started reviewing.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js [hunk 1]
```diff
@@ -1629,6 +1629,18 @@ AUI.add(
 						'#' + instance.getInputName() + 'Title'
 					);
 
+					var parsedAssetTitleMap = instance.getParsedValue(
+						parsedValue.assetTitleMap
```
Comment (orsolyalukacs) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js:1633]: This one should be `parsedValue.assetTitle`

Comment (dacousalr) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js:1633]: I have renamed the former JSON attribute assetTitle to assetTitleMap in line 1809 so it should be parsedValue.assetTitleMap as is.

Comment (orsolyalukacs) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js:1633]: Sorry, I had the old code in 1809 cached in the browser for some reason, even after I clean deployed. You are right, it's working as it is! :)

Comment (dacousalr) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js:1633]: No problem :)

**Comment by orsolyalukacs** [comment]
Pull request submitted:
https://github.com/dacousalr/liferay-portal/pull/145

**Pull request closed without merging**